ngraph/core/include/openvino/core/type.hpp

```diff
@@ -7,14 +7,17 @@
 #include <cstdint>
 #include <cstring>
 #include <functional>
+#include <iostream>
 #include <memory>
 #include <string>
 #include <utility>
 #include <vector>
 
+#include "ngraph/compatibility.hpp"
```

Comment:
should not it be a part of new API?

Reply:
I don't want to include new headers to old API. It is why I introduced this header in old structure.
`type_info` is used only in old API.